README.md

```diff
@@ -471,17 +473,18 @@ In this section I've collected some learnings that I have found relevant and use
 
 ### Limited Liability Companies & Dividends
 
-As the owner and primary shareholder of a limited liability company, you can pay yourself dividends. These dividends, if they are below a certain threshold amount, will be favourably taxed at 20% of the amount. If you exceed the threshold amount, the excess will be taxed as income, which generally has a much higher taxation rate. 
+As the owner and primary shareholder of a limited liability company, you can pay yourself dividends. These dividends, if they are below a certain threshold amount, will be favourably taxed at 20% of the amount. If you exceed the threshold amount, the excess will be taxed as income, which generally has a much higher taxation rate. Dividends are signficantly lower taxed than the equivalent salary above the limit for state tax (brytpunkt för statlig skatt) and so if you salary is already above this limit then you will benefit hugely after tax by paying dividends instead. It is one of the major advantages of freelancing over being a consultant/employee for someone else.
```

Comment:
"Dividends are significantly lower taxed compared to the equivalent salary above the limit for state tax (brytpunkt för statlig skatt). If this is your case, then you will benefit hugely after tax by paying dividends instead. It is one of the major advantages of freelancing over being a consultant/employee for someone else."

Reply:
I think this is a bit unclear - AFAIK dividends are lower taxed than both salary below and above the limit for state tax. 

```suggestion
As the owner and primary shareholder of a limited liability company, you can pay yourself dividends. These dividends, if they are below a certain threshold amount, will be favourably taxed at 20% of the amount. If you exceed the threshold amount, the excess will be taxed as income, which generally has a much higher taxation rate. 

As a rule of thumb, as long as you are within the threshold amount, it is preferable to increase the dividend payout rather than to increase your salary, and is one of the financial benefits of having your own company over being employed elsewhere. 
```

WDYT?

Reply:
Below the limit for state tax is a bit harder to quantify as you get direct benefits from that tax in the form of SGI and pension contributions.